content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -225,3 +225,10 @@ One use case for selecting over labels is to constrain the set of nodes onto whi
 See the documentation on [node selection](/docs/concepts/configuration/assign-pod-node/) for more information.
 
 {{% /capture %}}
+
+{{% capture whatsnext %}}
+* Learn more about [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
+* Learn more about [different ways to use labels](/docs/concepts/overview/working-with-objects/common-labels/#examples).
+* Learn more about [node selector](/docs/concepts/configuration/assign-pod-node/#nodeselector).
```

Comment:
Maybe change “node selector” to one of:
- `nodeSelector`
- assigning Pods to Nodes based on labels

@rajeshdeshpande02 do you prefer either of those wordings?